crates/ruff_server/src/session.rs

```diff
@@ -35,7 +38,7 @@ pub struct Session {
 /// a specific document.
 pub struct DocumentSnapshot {
     resolved_client_capabilities: Arc<ResolvedClientCapabilities>,
-    client_settings: settings::ResolvedClientSettings,
+    client_settings: settings::ClientSettings,
```

Comment:
Somewhat scary that we clone the settings for every request! We should probably use an `Arc` here

Reply:
I fixed this